.buildkite/hooks/pre-command

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-eval "$(ejson2env .buildkite/env/secrets.ejson)"
```

Comment:
I think we don't use it for a long time. I comment out this line at this PR. I'm going to have another PR to clean this old mechanism